src/Main.elm

```diff
@@ -69,17 +41,15 @@ arrivalBarGraph : Spec
 arrivalBarGraph =
     let
         des =
-            description "A simple bar chart with embedded data."
+            description "A visualization of arrival time averages for the MTA G Train Greenpoint Ave. stop"
 
         data =
-            dataFromColumns []
-                << dataColumn xAxisName (strs fakeArrivalTimes.day)
-                << dataColumn yAxisName (nums fakeArrivalTimes.meanArrivalDifferences)
+            dataFromUrl "http://localhost:4000/" []
 
         enc =
             encoding
-                << position X [ pName xAxisName, pOrdinal, pSort [ soCustom (strs fakeArrivalTimes.day) ], pAxis [ axTitleFontSize axisFontSize ] ]
-                << position Y [ pName yAxisName, pQuant, pAxis [ axTitleFontSize axisFontSize ] ]
+                << position X [ pName "day", pOrdinal, pSort [], pAxis [ axTitleFontSize axisFontSize ] ]
```

Comment:
If you're going to set the axis names directly as strings here, then do you still need the `xAxisName` and `yAxisName` variables above?

I do think that it's easier to read if you keep them as variable names and set the values separately.